blog/_posts/2022-02-14-preview-environments.md

```diff
@@ -0,0 +1,322 @@
+---
+slug: "preview-environments"
+date: 2022-02-14
+author: nick
+layout: blog
+title: "How I Built a Toy Preview Environment Platform and What Happened Next"
```

Comment:
This title is really a mouthful, plus it's almost completely duplicated in the subtitle.
Suggest to make it slightly more lightweight
I tried building / built a preview environment platform. Here's what happened

Reply:
ya, i totally agree. want to think about a more troll-y title. :thinking: :thinking: :thinking:

Reply:
A Preview Environment Preview: The Operator Strikes Back

Operators Previewing Preview Environments Preview Preview Environments

Reply:
Experiments With Preview Environments (And Making Security Professionals Cringe)

Preview Environments As Dev Environments As Code

Experiments in `kink`: Using Kubernetes-in-Kubernetes To Create Preview Environments